### PR TITLE
Split out AWS service accounts in osd-serviceaccounts

### PR DESCRIPTION
**Pull request by jewzaam**
It is failing to apply this SSS on GCP because of missing namespaces.

### CODE REVIEW

**Review by jharrington22 — APPROVED** [review]
/lgtm

File: deploy/osd-serviceaccounts/aws/00-serviceaccounts.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-ingress-operator
```
Comment (cblecker) [inline deploy/osd-serviceaccounts/aws/00-serviceaccounts.yaml:5]: CIO is deployed on GCP too

**Review by cblecker — CHANGES_REQUESTED** [review]
/hold

**Comment by jewzaam** [comment]
Fixed CIO SA and the config for the SA SSS.  Somehow I moved, didn't copy, to the new subdir :facepalm:

**Review by cblecker — APPROVED** [review]
/hold cancel

**Pull request merged**